engine/engine/src/engine.cpp

```diff
@@ -1466,294 +1425,297 @@ namespace dmEngine
         return memcount;
     }
 
-    void Step(HEngine engine)
+    static void StepFrame(HEngine engine, float dt)
```

Comment:
The code here is the old contents of Engine::Step()
As evident, a lot of the stuff isn't really something we'd like to run more than once per engine frame.
(e.e HID update, Flip() etc) but for now this is what we have.